src/settings/NotificationsScreen.js

```diff
@@ -11,19 +13,82 @@ import SwitchRow from '../common/SwitchRow';
 import Screen from '../common/Screen';
 import * as api from '../api';
 import ServerPushSetupBanner from '../common/ServerPushSetupBanner';
+import NestedNavRow from '../common/NestedNavRow';
+import { useAppState } from '../reactNativeUtils';
+import { IconAlertTriangle } from '../common/Icons';
+
+const {
+  ZLPConstants,
+  Notifications, // android
+  ZLPNotifications, // ios
+} = NativeModules;
```

Comment:
I think ideally we would arrange things so that this platform-dependence was a bit more encapsulated: either giving these two native modules interfaces that are more like each other, or having a small JS module that consumes them both and presents a crisper and more unified interface, or both.

But that needn't come before merging this PR. If we find ourselves investing further in making native modules and in particular in doing so on both platforms, we can refactor this as part of that.

Reply:
Yeah, these names were not satisfying to write in this JS file. 🙂

> But that needn't come before merging this PR. If we find ourselves investing further in making native modules and in particular in doing so on both platforms, we can refactor this as part of that.

Yeah, that was my feeling too.